faqrenderer/faq.md

```diff
@@ -0,0 +1,121 @@
+
+| Language | Old version | New version |
+| - | - | - |
+| Python | 3.6 | 3.7.8 |
+| R | 3.6.1 | 4.0.2 |
+| Julia | 1.1.0 | 1.5.0 |
+| Octave | 4.2.2 | 4.2.2 |
+| C++ (cling) | 0.6 | 0.6 |
+| SageMath | 8.1 | 9.1 |
+
+Most of the packages included in this New Default Environment are at their latest version as of August 2020. Compared to the legacy environment, a few packages have been removed in this new environment, which are listed below. 
+
+- pymol and ipymol: Requires extra conda channels and is not free software. Can be installed manually in a new conda environment.
+- opty: Requires Python <=3.6. Can be installed manually in a new conda environment with Python 3.6.
+- rpy2: Requires R <= 4.0. Can be installed manually in a new conda environemt with R 3.6.
+
+If you require the above packages or older versions of existing ones, you should use the legacy environment or [create a conda environment](#how-do-i-create-a-conda-environment) with your custom package versions. Please email us at [jupyterteam@ucdavis.edu](mailto:jupyterteam@ucdavis.edu) if you have concerns or issues with the new environment.
+
+## How do I access RStudio?
+
+To open the RStudio interface, you can select the RStudio notebook icon from the Jupyter Launcher (Ctrl+Shift+L). This will open a new JupyterLab tab which has an RStudio interface and access to all the same packages you would find elsewhere on your account.
+
+![RStudio Launcher icon](RStudio-launcher.png)
+
+## How can I install custom packages?
+
+You are free to install any additional packages available through conda by [creating a conda environment](#how-do-i-create-a-conda-environment), or by running [conda install 'package'](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/installing-with-conda.html) commands within your custom environment. These packages will persist for you. You may also use `conda install` directly in the `(notebook)` environment provided by default, but these packages won't be permanent and will be wiped anytime your server restarts.
+
+For packages not available through conda or for languages which have their own package manager such as SageMath and Julia, we suggest that you [email us](mailto:jupyterteam@ucdavis.edu) to provide them for you by default in the environment. Packages must be installed to the `/home/jovyan/` directory to persist and Jula and SageMath do not install there by default. For Instructors wanting to use custom packages for their class, [you'll probably want to email us as well](#how-do-i-set-up-custom-environments-for-my-class).
```

Comment:
You need to state here that we will only install packages from Ubuntu 20.4 (or 18.04 if we are still on that) APT repositories https://packages.ubuntu.com/ or from conda-forge https://conda-forge.org/feedstocks/. Anyone can submit packages to both systems: https://www.debian.org/doc/manuals/distribute-deb/distribute-deb.html and https://conda-forge.org/#contribute. Once packages are available there, email us with the package names and we will install them.

Any other packages must be installed individually by the user using the recommended installation process of the package.

Reply:
Currently we install Julia packages outside of both APT and conda using Project.toml. I'm not precisely sure how we could install additional SageMath packages (of if that would ever be necessary, since it installs a ton of stuff for itself), but we install the language itself via a postBuild file. Should I instead just say that the only APT packages we can support are for bionic 18.04 (our verison), and conda packages must be thru conda-forge? Of course, also mentioning how to submit to these.